include/librealsense2/hpp/rs_processing.hpp

```diff
@@ -512,9 +512,9 @@ namespace rs2
     {
     public:
         /**
-        * Creates YUY decoder processing block. This block accepts raw YUY frames and outputs frames of other formats.
-        * YUY is a common video format used by a variety of web-cams. It benefits from packing pixels into 2 bytes per pixel
-        * without signficant quality drop. YUY representation can be converted back to more usable RGB form,
+        * Creates y411 decoder processing block. This block accepts raw y411 frames and outputs frames of other formats.
```

Comment:
Drop a link to the spec for Y411.
State the issue we see in Windows with the frame size.
State how this is exposed in the FW as 'NV12'.